kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -253,11 +253,14 @@
     },
     watch: {
       // Update facilities whenever a watched task completes
-      facilityTasks(newTasks) {
+      facilityTasks(newTasks, prevTasks) {
         for (const index in newTasks) {
           const task = newTasks[index];
           if (this.taskIdsToWatch.includes(task.id)) {
-            if (task.status === TaskStatuses.COMPLETED) {
+            if (
+              task.status === TaskStatuses.COMPLETED ||
```

Comment:
We werent taking into account that repeating tasks doesnt have a "completed" status when they are synced and thats why we hadnt the `last_successful_sync` value updated.